src/store/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, applyMiddleware } from "redux";
+import reduxImmutableStateInvariant from "redux-immutable-state-invariant";
+
+import rootReducer from "../reducers";
+
+export default function configureStore(initialState) {
+  return createStore(
+    rootReducer,
+    initialState,
+    applyMiddleware(reduxImmutableStateInvariant())
```

Comment:
`reduxImmutableStateInvariant` is best used for development and not production so could you have a check on which environment one is at before adding it.
It reduced apps perfomance and thats why its only used on development, check on this [link](https://github.com/leoasis/redux-immutable-state-invariant) for more and  on usage.

Reply:
I added store configurations for development and production environment.